Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -89,7 +89,7 @@ tasks:
       id: 0f808daf-5e80-401e-8c0e-22a9abe1e5ac
       version: -1
       name: 'Manual – Search possible hosts'
-      description: "Dear Analyst,\n\nEnsure that the 'XQL Query Engine' integration is active. If it's not enabled, activate the integration or manually run the following XQL query to determine if a file was successfully downloaded from a malicious URL using certutil.\n\nQuery:\n\nconfig timeframe=7d | dataset = xdr_data | filter event_type = ENUM.FILE and lowercase(actor_process_image_name) = \"jstart.exe\" and lowercase(action_file_path) contains  \"visualcomposer\" | fields  agent_id,agent_hostname,agent_ip_addresses | dedup agent_id  \n\nIf the query output indicates on possible activity with file/s, investigate those files and quarantine/remove them per your need."
+      description: "Dear Analyst,\n\nEnsure that the 'XQL Query Engine' integration is active. If it's not enabled, activate the integration or manually run the following XQL query to determine if any hosts are running a possibly vulnerable product.\n\nQuery:\n\nconfig timeframe=7d | dataset = xdr_data | filter event_type = ENUM.FILE and lowercase(actor_process_image_name) = \"jstart.exe\" and lowercase(action_file_path) contains  \"visualcomposer\" | fields  agent_id,agent_hostname,agent_ip_addresses | dedup agent_id"
```

Comment:
i'd consider changing to 'a potentially vulnerable product'

Reply:
Done